homework/index.js

```diff
@@ -28,9 +30,37 @@
     });
     return elem;
   }
-
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    // eslint-disable-next-line prefer-template
+    const li = createAndAppend('li', ul, { id: 'item' });
+    const repoName = createAndAppend('p', li);
+    repoName.innerHTML = "<span class = 'nameItem'>Repository:</span> ";
+    createAndAppend('a', repoName, {
+      href: repo.html_url,
+      text: repo.name,
+    });
+    const repoDescription = createAndAppend('p', li, { class: 'paragraph' });
+    if (repo.description) {
+      repoDescription.innerHTML =
+        "<span class = 'nameItem'>Description:</span> " + repo.description;
+    } else {
+      repoDescription.innerHTML =
+        "<span class = 'nameItem'>Description: </span> N/A";
+    }
+    const repoForks = createAndAppend('p', li, { class: 'paragraph' });
+    repoForks.innerHTML =
+      "<span class = 'nameItem'>Fork: </span>" + repo.forks_count;
+    const repoDate = createAndAppend('p', li);
+    repoDate.innerHTML =
+      "<span class = 'nameItem'>Updated: </span> " +
+      new Date(repo.updated_at).toLocaleString();
+  }
```

Comment:
I think using an HTML table would produce a nice UI. It would vertically align all repository values and also ensure that longer values do not wrap to the extreme left margin as happens in your case if you decrease the screen width:

![image](https://user-images.githubusercontent.com/2788771/65370300-88fc1c80-dc57-11e9-8696-54e55aa04828.png)

Reply:
I did :+1: